docs/dependency-specification.md

```diff
@@ -74,10 +74,13 @@ Here are some examples of inequality requirements:
 ### Exact requirements
 
 You can specify the exact version of a package.
+
+`==1.2.3` is an example of an exact version specification.
+
 This will tell Poetry to install this version and this version only.
 If other dependencies require a different version, the solver will ultimately fail and abort any install or update procedures.
 
-#### Multiple requirements
+### Multiple requirements
```

Comment:
Wait, I thought that it was incorrectly indented, because it does not relate at all to `Exact requirements` section, but re-reading the doc, the fact that it's an H4 may be correct, and it should just be moved under [Inequality requirements](https://python-poetry.org/docs/dependency-specification/#inequality-requirements) instead.

Reply:
Added https://github.com/python-poetry/poetry/pull/5874/commits/53374dfff4ac7ce05cc87ed9e9b3006069c232a7 to change that, though I would prefer to have confirmation that this is the correct thing to do.